### ISSUE TITLE
AttributeError: 'HTTPResponse' object has no attribute 'readall'

### ISSUE THREAD

**Issue by KCC13** [bveliqi/adversarial-vision-challenge#8]
Hello, 

I was trying to run `avc-test-model .`

but was faced with the error:  AttributeError: 'HTTPResponse' object has no attribute 'readall'.

I think the problem is that urllib in python >= 3.5 do not support `readall()`,

but in the "avc-test-setup", `readall()` would be used in all python3 versions:

```
if sys.version_info[0] == 3:
	data = json.loads(request.urlopen('https://pypi.python.org/pypi/adversarial-vision-challenge/json').readall().decode('utf-8'))
```

**Comment by jonasrauber** [comment]
Thanks, we are aware and the master branch should already fix this (if you install from github), but this is still untested (we welcome your feedback). We will release an updated version on PyPI once we have tested it (due to some technical issues, this takes a little bit longer than usual).

**Comment by jonasrauber** [comment]
This is fixed in 1.0.0
